recipe/meta.yaml

```
@@ -23,7 +25,7 @@ requirements:
     - pkg-config
     - make
   host:
-    - fftw
+    - fftw >=3
```

As this is a shared library, this will break the version pinning when rendering for the reasons I wrote in this comment: https://github.com/conda-forge/root-feedstock/pull/68#issuecomment-558266527
```suggestion
    - fftw
```

---

Also, you should try to renderer after changing dependencies as sometimes something different is needed.